chef/cookbooks/hyperv/recipes/do_ceilometer.rb

```diff
@@ -0,0 +1,8 @@
+raise if not node[:platform] == 'windows'
+
+include_recipe "hyperv::openstack_install_ceilometer"
+include_recipe "hyperv::openstack_install_neutron"
+include_recipe "hyperv::config_neutron"
+include_recipe "hyperv::config_ceilometer"
+include_recipe "hyperv::register_services_ceilometer"
+include_recipe "hyperv::register_services_neutron"
```

Comment:
Hm, right, OK. I think it might be good to change barclamp-ceilometer to include such dependency, but let's keep it somple now.